src/dataflow/node.h

```diff
@@ -144,50 +147,56 @@ struct Node {
 
   // Helpers
 
-  void addValue(Node* value) {
-    values.push_back(value);
-  }
-  Node* getValue(Index i) {
-    return values.at(i);
-  }
+  void addValue(Node* value) { values.push_back(value); }
+  Node* getValue(Index i) { return values.at(i); }
 
   // Gets the wasm type of the node. If there isn't a valid one,
   // return unreachable.
   wasm::Type getWasmType() {
     switch (type) {
-      case Var:   return wasmType;
-      case Expr:  return expr->type;
-      case Phi:   return getValue(1)->getWasmType();
-      case Zext:  return getValue(0)->getWasmType();
-      case Bad:   return unreachable;
-      default:    WASM_UNREACHABLE();
+      case Var:
+        return wasmType;
+      case Expr:
+        return expr->type;
+      case Phi:
+        return getValue(1)->getWasmType();
+      case Zext:
+        return getValue(0)->getWasmType();
+      case Bad:
+        return unreachable;
+      default:
+        WASM_UNREACHABLE();
     }
   }
 
   bool operator==(const Node& other) {
-    if (type != other.type) return false;
+    if (type != other.type)
+      return false;
     switch (type) {
       case Var:
-      case Block: return this == &other;
+      case Block:
+        return this == &other;
       case Expr: {
         if (!ExpressionAnalyzer::equal(expr, other.expr)) {
           return false;
         }
         break;
       }
-      case Cond:  if (index != other.index) return false;
+      case Cond:
+        if (index != other.index)
+          return false;
       default: {}
     }
-    if (values.size() != other.values.size()) return false;
+    if (values.size() != other.values.size())
+      return false;
     for (Index i = 0; i < values.size(); i++) {
-      if (*(values[i]) != *(other.values[i])) return false;
+      if (*(values[i]) != *(other.values[i]))
+        return false;
```

Comment:
ifs with the body on another line and no `{ }` are quite bad in my opinion. but i guess clang-format can't add braces since it just adds whitespace?

Reply:
I've got used to this in llvm/lld.. in lld we even have nested for/while/if all with single line bodies:

```
for (...) 
   if (..) 
       do_something();
```

In fact its bad style to add curly braces in the above case in that codebase.   Just another data point I guess.

Reply:
Yes, as I said in the previous comment, clang-format does not add any characters except for whitespaces and `//` for splitting comments to multiple lines.

Reply:
@sbc100 I don't mind not having curly braces either in single line ifs. But if more people think we should have it in binaryen codebase, we can come up with .clang-tidy that specifies those behaviors and apply it to the whole codebase as a follow-up patch.